homeassistant/components/influxdb/const.py

```diff
@@ -135,6 +138,8 @@
     vol.Optional(CONF_PATH): cv.string,
     vol.Optional(CONF_PORT): cv.port,
     vol.Optional(CONF_SSL): cv.boolean,
+    vol.Optional(CONF_PRECISION): cv.string,
+    vol.Optional(CONF_RESEND_PERIOD): vol.Coerce(float),
```

Comment:
We should not add the resend feature to influxdb. That's not this integration's responsibility. This integration should just consume data generated by other integrations.

As said above, each integration can decide to force state updates if needed.

Please remove the resend feature and focus this PR on the precision feature.